lib/tasks.js

```diff
@@ -130,8 +130,12 @@ const runTasks = async (opts, injected = {}) => {
       github.release && github.releaseNotes && logPreview(log, 'release notes', await ghClient.getNotes());
       const ghRelease = () => ghClient.release({ version, isPreRelease, changelog });
       const ghUploadAssets = () => ghClient.uploadAssets();
+      const ghPublishRelease = () => ghClient.maybePublishRelease();
```

Comment:
Maybe with `maybePublishRelease` it makes sense to rename `release` to `draftRelease`?